bot/plugins/help.py

```diff
@@ -0,0 +1,57 @@
+import crescent
+import hikari
+
+from bot.app import Plugin
+from bot.constants import EMBED_COLOR
+
+plugin = Plugin()
+
+HELP_EMBEDS = [
+    hikari.Embed(
+        description=(
+            "Hi! My name is io, and my job is to run code."
+            "\n I can run any code inside of code blocks:"
+            "\n\\`\\`\\`<language-name>"
+            "\n<your code here>"
+            "\n\\`\\`\\`"
+        ),
+        color=EMBED_COLOR,
+    ),
+    hikari.Embed(
+        description=(
+            "\n\\* Running code - Start your message with `io/run`."
+            "\n\\* View Assembly - Start your message with `io/asm`."
+            "\n"
+            "\nYou can use message commands by right-clicking on a message, "
+            "selecting the `Apps` subcategory, then clicking on the `Create Instance` "
+            "command."
+        ),
+        color=EMBED_COLOR,
+    ),
+]
+
+
+@plugin.include
+@crescent.command
+async def help(ctx: crescent.Context):
+    await ctx.respond(embeds=HELP_EMBEDS)
+
+
+@plugin.include
+@crescent.event
+async def on_message(message: hikari.MessageCreateEvent):
+    me = plugin.app.get_me()
+
+    if not me:
+        return
+
+    if message.author_id == me.id:
+        return
+
+    if not (
+        message.content == me.mention
+        or message.content.removeprefix(me.mention).strip() == "help"
+    ):
+        return
+
+    await message.message.respond(embeds=HELP_EMBEDS)
```

Comment:
Doesn't this mean that saying "help" will trigger the bot? I think that commands besides io/run and io/asm should be slash commands. So @mention the bot (with no content), or use /help.

Reply:
fixed